person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
Please make sure you set these variables in the exact same order you do in the parameters in your constructor as it is considered best practice.